### DISCUSSION TITLE
Do the phases matter?

### DISCUSSION THREAD

**Discussion by thehounddog** [emporia-vue-local/esphome#41] [Q&A]
I have three phase, but I was planning on just using the black power supply to power the unit from a power point I have handy, and just use a single reference voltage. This is just for monitoring individual circuits on a sub board, not for whole house supply, so I'm happy with approximate power. I have configured all circuits to just use just phase A, but I'm seeing some results fluctuating between positive and negative. So - are the three wires just for reference voltage or does the phase matter too?

**Comment by flaviut** [comment]
You definitely need to match voltage & current phases to get good power data. Remember how 3-phase works:

![3_phase_AC_waveform svg](https://user-images.githubusercontent.com/5213469/195077194-a5679879-66a4-458b-8a10-45cd9edb08cf.png)

If your phase 2 current is at 0.7A, and you're using phase 1 voltage, then the power is 0.7A×0V=0W, and that's not right. The Vue doesn't show you this level of precision, millisecond-by-millisecond, but internally, this is how it works.

**Reply by thehounddog** [reply]
Yes I had confirmed this by moving the black wire to different phases and it cleaned up the results on the circuits that matched that phase. I just thought that if I specified all circuits as phase A in the config the voltage would never be 0v because it should come from the black wire. Obviously works different to a single clamp meter I made using ESP8266 which just measures amps, which I then multiply by a reference voltage I read in from somewhere else. This is feeling above my pay grade. I need to get an electrician to wire in each phase properly.
